api/tests/unit/sse/test_tasks.py

```diff
@@ -121,10 +121,30 @@ def test_track_sse_usage(
         [
             call("sse_call"),
             call().field("request_count", 2),
-            call().field().tag("organisation_id", environment.project.organisation_id),
-            call().field().tag().tag("project_id", environment.project_id),
-            call().field().tag().tag().tag("environment_id", environment.id),
-            call().field().tag().tag().tag().time(second_access_log.generated_at),
+            call().field().tag("environment_id", environment.id),
+            call().field().tag().tag("project_id", environment.project.id),
+            call().field().tag().tag().tag("project", environment.project.name),
+            call()
+            .field()
+            .tag()
+            .tag()
+            .tag()
+            .tag("organisation_id", environment.project.organisation.id),
+            call()
+            .field()
+            .tag()
+            .tag()
+            .tag()
+            .tag()
+            .tag("organisation", environment.project.organisation.name),
+            call()
+            .field()
+            .tag()
+            .tag()
+            .tag()
+            .tag()
+            .tag()
+            .time(second_access_log.generated_at),
```

Comment:
This is ugly as heck but I'm not sure if there are any other options. I tried doing _something_ like this but to no avail: 

```python
mocked_influx_point.assert_called_once_with(
    call("see_call").field("request_count", 2).tag("environment_id", environment.id)...
)
```

Reply:
Haha yeah, it is a weird one